__device-tests__/gutenberg-editor-sanity-test-1-visual.test.js

```diff
@@ -0,0 +1,294 @@
+/**
+ * External dependencies
+ */
+import wd from 'wd';
+
+/**
+ * Internal dependencies
+ */
+const { blockNames } = editorPage;
+import { takeScreenshot } from './utils';
+const { toggleOrientation, isAndroid, swipeFromTo } = e2eUtils;
+
+describe( 'Gutenberg Editor - Test Suite 4', () => {
+	describe( 'Columns block', () => {
+		it( 'displays placeholders when unselected', async () => {
+			await editorPage.addNewBlock( blockNames.columns );
+			// Wait for the modal to open
+			await editorPage.driver.sleep( 3000 );
+			// Dismiss coluns layout picker
+			await editorPage.driver
+				.elementByAccessibilityId( 'Cancel' )
+				.click();
+			// Wait for the modal to close
+			await editorPage.driver.sleep( 3000 );
+			// Select title to unfocus the block
+			const titleElement = await editorPage.getTitleElement();
+			await titleElement.click();
+			await editorPage.dismissKeyboard();
+
+			// Visual test check for portrait orientation
+			let screenshot = await takeScreenshot();
+			expect( screenshot ).toMatchImageSnapshot();
+
+			await toggleOrientation( editorPage.driver );
+			// Wait for the device to finish rotating
+			await editorPage.driver.sleep( 3000 );
+
+			// Visual test check for landscape orientation
+			screenshot = await takeScreenshot();
+			expect( screenshot ).toMatchImageSnapshot();
+
+			await toggleOrientation( editorPage.driver );
+			// Wait for the device to finish rotating
+			await editorPage.driver.sleep( 3000 );
+			const columnsBlock = await editorPage.getBlockAtPosition(
+				blockNames.columns
+			);
+			await columnsBlock.click();
+			await editorPage.removeBlock();
+		} );
+
+		it( 'displays correctly in portrait and landscape orientations', async () => {
+			await editorPage.addNewBlock( blockNames.columns );
+			// Wait for the modal to open
+			await editorPage.driver.sleep( 3000 );
+			// Dismiss coluns layout picker
+			await editorPage.driver
+				.elementByAccessibilityId( 'Cancel' )
+				.click();
+			// Wait for the modal to close
+			await editorPage.driver.sleep( 3000 );
+
+			// Click the block appender within the first column
+			await editorPage.driver
+				.elementByAccessibilityId( 'Column Block. Row 1' )
+				.click()
+				.click();
+
+			// Click the Columns block type button
+			const blockButton = await editorPage.findBlockButton(
+				blockNames.columns
+			);
+			if ( isAndroid() ) {
+				await blockButton.click();
+			} else {
+				await editorPage.driver.execute( 'mobile: tap', {
+					element: blockButton,
+					x: 10,
+					y: 10,
+				} );
+			}
```

Comment:
I wonder if we could update the `addNewBlock` helper, to be able to skip some steps if the Inserter is already visible. I mean being able to pass some options to it, so we could reuse it for this case and future blocks like `Group`, for example. What do you think?